jobs: use the github plugins instead of snappy-tests-job to get the branc

elopio opened the pull request.

fgimenez commented:
The shell script for executing the jobs is great, it would be awesome if we could reuse it somehow. Maybe adding a execute-ci.sh file in the source code of the project and letting the jobs just call it, or use this plugin [1]

Why are you removing the snappy-cloud-test job? It's the downstream job triggered after a new image has been built, the first one that exercises it. Actually the daily ones could be replaced by this one.

[1] https://wiki.jenkins-ci.org/display/JENKINS/Managed+Script+Plugin

elopio commented:
My fear about reusing it is that it will turn in snappy-tests-job, with tons of flags that we need to constantly extend. I would prefer to reduce it to a handful of scripts. First, we'll remove three statements with ubuntu-core/snappy/465. Then, we can modify integration-tests/main.go to generate the xml. I was thinking about how to reduce the testbed creation and delete to one statement, but I have no ideas there. Instead of having jenkins managing this scripts, I would like us to put them in the repo in .jenkins or something like that. I don't have a clear idea about how to specify the multiple scenarios that we are running here, though.

I removed snappy-cloud-test because I can't clone with the plugin master and 15.04 on the same job. As we already have the dailies that do the same, I just added the channel parameter to them.

fgimenez commented:
Ok, we can begin with this and refactor later. Sorry, didn't notice the changes in the jobs triggered by the create jobs, LGTM
